packages/amplify_api/example/lib/graphql_api_view.dart

```diff
@@ -76,8 +79,9 @@ class _GraphQLApiViewState extends State<GraphQLApiView> {
       }
     }''';
 
-    var operation = await Amplify.API
-        .query<String>(request: GraphQLRequest(document: graphQLDocument));
+    var operation = await Amplify.API.query<String>(
+        request: ModelQueries.get(
+            Blog.classType, "f70d1142-12da-4564-a699-966a75f96db6"));
```

Comment:
Fetching by a specific ID will not work for other folks in the example app, may need to rethink how these helpers are integrated in the example app. I think it will be nice to have this in example app at some point as a showcase, but it might make sense to exclude example app changes from this PR to keep it more focused.

Reply:
For sure, I think we can maintain the original spec once we have the `list` method implemented. I've been using this for testing and its should be ripped out before production. I'll drop a comment saying as much too.